homeassistant/components/vorwerk/translations/de.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Files in the translations folder should not be included in the PR, they will be automatically generated from Lokalise and `strings.json`

Reply:
Good to know. Should I remove the German translation as well?